packages/common/src/utils/challenges.ts

```diff
@@ -417,7 +417,11 @@ export const getChallengeStatusLabel = (
       }
       return 'No Recent Activity'
     case ChallengeName.FirstWeeklyComment:
-      if (challenge.disbursed_amount && !challenge.claimableAmount) {
+      if (
+        challenge.disbursed_amount &&
```

Comment:
can we extract to a named var

Reply:
could but since it's in common would be hard to share w/ web i think. like would have to add a whole config thing for it which i am loathe to do.